packages/renderer/src/lib/ui/FormPage.svelte

```diff
@@ -25,5 +28,19 @@ export function goToPreviousPage(): void {
   <slot slot="icon" name="icon" />
   <slot slot="actions" name="actions" />
   <slot slot="tabs" name="tabs" />
-  <slot slot="content" name="content" />
+  <slot slot="content">
+    {#if contentLayout === 'engine'}
+      <div class="p-5 min-w-full h-full">
+        {#if showEmptyScreen}
+          <NoContainerEngineEmptyScreen />
+        {:else}
```

Comment:
I do not see why we need to have a 'layout' enum property inside this component

it's not this component that should do the change and display a nocontainerengine but the callee/parent

Reply:
I put this here bc the FormPage is already a wrapper for the FormPage from the ui package, so this is really tight to podman desktop and its needs. There are multiple components using the same emptyscreen/content pattern so i thought that the formPage could have different layouts and handle it.
Do you prefer every single component to handle its own css/empty screen and leave the formPage as it was?

Reply:
This is some small code savings but feels odd to me as well - saying 'engine' changes the layout, and we have a fixed empty screen but only show it based on a reason the page has to provide. Maybe it'd work better as an EngineFormPage that extends FormPage and automatically shows the empty screen when needed? (or whenever the slot isn't provided?) It currently feels more like 'hard-coded requirements for specific children' than 'optional feature' like tabs or breadcrumb.

Reply:
Added an EngineFormPage component